.taplo.toml

```diff
@@ -0,0 +1,19 @@
+include = ["**/Cargo.toml"]
+
+[formatting]
+align_entries = false
+align_comments = false
+array_trailing_comma = true
+array_auto_expand = true
+array_auto_collapse = true
+compact_arrays = true
+compact_inline_tables = false
+compact_entries = false
+column_width = 80
+indent_tables = false
+indent_entries = false
+indent_string =  "	"
```

Comment:
I think it's better to use space here, maybe 2 or 4 spaces, but not tabs

Reply:
👌